pkg/plugins/manager/fakes/fakes.go

```diff
@@ -301,6 +301,7 @@ func (m *FakeProcessManager) Stop(ctx context.Context, pluginID string) error {
 type FakeBackendProcessProvider struct {
 	Requested map[string]int
 	Invoked   map[string]int
+	RejectID  string
```

Comment:
Nit:  Would be nice for this fake to follow the pattern of the other fakes where it can include a func override. For example:

```go
type FakeBackendProcessProvider struct {
	Requested          map[string]int
	Invoked            map[string]int
    BackendFactoryFunc func(_ context.Context, p *plugins.Plugin) backendplugin.PluginFactoryFunc
```

So that your `TestLoader_Load_SkipUninitializedPlugins` test can just specify:

```go
fakeProvider := &FakeBackendProcessProvider{
    BackendFactoryFunc: func(_ context.Context, _ *plugins.Plugin) backendplugin.PluginFactoryFunc {
        return func(_ string, _ log.Logger, _ []string) (backendplugin.Plugin, error) {
		       return nil, errors.New("rejected")
       }
    }
}
```